velox/core/QueryConfig.h

```diff
@@ -359,6 +359,15 @@ class QueryConfig {
   static constexpr const char* kDriverCpuTimeSliceLimitMs =
       "driver_cpu_time_slice_limit_ms";
 
+  /// The max bytes number of normalized keys for each row in the prefix-sort.
+  /// The default value is max int_64, and set to 0 to disable prefix-sort.
+  static constexpr const char* kPrefixSortNormalizedKeyMaxBytes =
+      "prefixsort_normalized_key_max_bytes";
+
+  /// The minimum number of rows that can trigger the prefix-sort. This is set
```

Comment:
``
Minimum number of rows to use prefix-sort. The default value has been derived using micro-benchmarking.
```